sdk/keyvault/keyvault-secrets/src/transformations.ts

```diff
@@ -23,9 +23,13 @@ export function getSecretFromSecretBundle(
     value: secretBundle.value,
     name: parsedId.name,
     properties: {
-      expiresOn: (attributes as any).expires,
-      createdOn: (attributes as any).created,
-      updatedOn: (attributes as any).updated,
+      expiresOn: attributes?.expires,
+      createdOn: attributes?.created,
+      updatedOn: attributes?.updated,
+      enabled: attributes?.enabled,
+      notBefore: attributes?.notBefore,
+      recoverableDays: attributes?.recoverableDays,
+      recoveryLevel: attributes?.recoveryLevel,
```

Comment:
Woah! Thank you so much!

Before we had these transformers we used to use the spread operator a lot, and in my attempt to gain better control over it, I ended up missing these properties I think.

Should we see if we're having a similar issue in the other Key Vault clients?

Reply:
Good idea! Maybe we can merge this in and then I can separately check the other packages?

Reply:
Sounds good! I'll approve!